src/templates/it/error.html

```diff
@@ -0,0 +1,24 @@
+{% extends "base/error.html" %}
+
+{% block unknown_error %}Errore sconosciuto{% endblock %}
+
+{%
+  set localizedStatusCode = {
+    400: "Bad Request",
+    404: "Pagina non trovata",
+    500: "Errore interno del server",
+    502: "Errore temporaneo del server"
+  }
+%}
+{% macro expandedTitle(errorTitle) %}Errore: {{ errorTitle }}{% endmacro %}
+{%
+  set localizedErrorMessage = {
+    "Unsupported year requested": "Anno richiesto non supportato",
+    "Unsupported chapter requested": "Capitolo richiesto non supportato",
+    "Unsupported language requested": "Lingua richiesta non supportata",
+    "Not Found": "L'URL richiesto non è stato trovato sul server. Se hai inserito manualmente l'URL, controlla l'ortografia e riprova."
+  }
+%}
+{% macro expandedErrorMessage(errorCode, errorDescription, errorMessage) %}
+{{ errorCode }} <i lang="it">{{ errorDescription }}</i>: {{ errorMessage }}
```

Comment:
```suggestion
{{ errorCode }} <i lang="en">{{ errorDescription }}</i>: {{ errorMessage }}
```

Reply:
How come?

Reply:
See comment here: https://github.com/HTTPArchive/almanac.httparchive.org/pull/1842#discussion_r555130077
 
> This was incorrect. The `errorDescription` is the server side on, in English (.e.g "Not Found)":
> 
> ![Example error message](https://user-images.githubusercontent.com/10931297/104201918-29181f00-5422-11eb-8a0f-42b24eb31d87.png)
> 
> If it was in Italian, we wouldn't need the lang attribute as the page level lang is set to it.
> 
> So we should revert this.

We show the actual server error (in English) as well as a translated version for the most common ones above. This handles the case where we've a weird server error that hasn't been translated so user at least gets something.